script/brave_license_helper.py

```diff
@@ -31,10 +32,120 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
+        # downloaded or only referenced upstream crates.
+        os.path.join('brave', 'third_party', 'rust', 'adler'),
+        os.path.join('brave', 'third_party', 'rust', 'aho_corasick'),
+        os.path.join('brave', 'third_party', 'rust', 'anyhow'),
+        os.path.join('brave', 'third_party', 'rust', 'base64'),
+        os.path.join('brave', 'third_party', 'rust', 'cfg_if'),
+        os.path.join('brave', 'third_party', 'rust', 'crc32fast'),
+        os.path.join('brave', 'third_party', 'rust', 'cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'cxxbridge_flags'),
+        os.path.join('brave', 'third_party', 'rust', 'cxxbridge_macro'),
+        os.path.join('brave', 'third_party', 'rust', 'getrandom', 'v0_2'),
+        os.path.join('brave', 'third_party', 'rust', 'hex'),
+        os.path.join('brave', 'third_party', 'rust', 'itoa', 'v1'),
+        os.path.join('brave', 'third_party', 'rust', 'lazy_static'),
+        os.path.join('brave', 'third_party', 'rust', 'libc'),
+        os.path.join('brave', 'third_party', 'rust', 'log'),
+        os.path.join('brave', 'third_party', 'rust', 'memchr'),
+        os.path.join('brave', 'third_party', 'rust', 'miniz_oxide'),
+        os.path.join('brave', 'third_party', 'rust', 'ppv_lite86'),
+        os.path.join('brave', 'third_party', 'rust', 'proc_macro2'),
+        os.path.join('brave', 'third_party', 'rust', 'quote'),
+        os.path.join('brave', 'third_party', 'rust', 'rand', 'v0_8'),
+        os.path.join('brave', 'third_party', 'rust', 'rand_chacha', 'v0_3'),
+        os.path.join('brave', 'third_party', 'rust', 'rand_core', 'v0_6'),
+        os.path.join('brave', 'third_party', 'rust', 'regex'),
+        os.path.join('brave', 'third_party', 'rust', 'regex_automata'),
+        os.path.join('brave', 'third_party', 'rust', 'regex_syntax'),
+        os.path.join('brave', 'third_party', 'rust', 'ryu'),
+        os.path.join('brave', 'third_party', 'rust', 'semver'),
+        os.path.join('brave', 'third_party', 'rust', 'serde'),
+        os.path.join('brave', 'third_party', 'rust', 'serde_derive'),
+        os.path.join('brave', 'third_party', 'rust', 'serde_json'),
+        os.path.join('brave', 'third_party', 'rust', 'static_assertions'),
+        os.path.join('brave', 'third_party', 'rust', 'syn'),
+        os.path.join('brave', 'third_party', 'rust', 'tokio'),
+        os.path.join('brave', 'third_party', 'rust', 'unicode_ident'),
+        os.path.join('brave', 'third_party', 'rust', 'winapi'),
+        os.path.join('brave', 'third_party', 'rust', 'wycheproof'),
+        os.path.join('brave', 'third_party', 'rust', 'zcash'),
+        os.path.join('brave', 'third_party', 'rust', 'zerocopy'),
+        os.path.join('brave', 'third_party', 'rust', 'zerocopy_derive'),
+
+        # Rust crates that do not supply license files.
+        os.path.join('brave', 'third_party', 'rust', 'bech32'),
+        os.path.join('brave', 'third_party', 'rust', 'blake2b_simd'),
```

Comment:
Similarly, this one has https://github.com/oconnor663/blake2_simd/blob/master/LICENSE. I'm not familiar with how we pull crates in, but would adding the LICENSE file to the sources array in BUILD.gn work?

Reply:
Same here, no LICENSE in the crate (https://static.crates.io/crates/blake2b_simd/blake2b_simd-1.0.1.crate). @bridiver suggested that we don't put LICENSE files into the crates.

Reply:
> suggested that we don't put LICENSE files into the crates.

Including the license text is actually a requirement of the MIT license (and most other licenses):

> "The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software."

Of course, it's not our responsibility when it comes to other people's crates, but for ours, we should include the license file.